### PR TITLE
Divyanshu | Test | Show Videos in the GIDDH panel in the software

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cu3buz1

### CODE REVIEW

File: apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    /**
+    * Initializes the component
+    *
+    * @memberof WatchVideoComponent
+    */
+    public ngOnInit(): void {
+        this.videoLink = VIDEOLINK[this.moduleName];
+        this.autoplay = this.autoplay ? 1 : 0;
+        this.videoLink = `${this.videoLink}&rel=0&autoplay=${this.autoplay}`;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts]:
```suggestion
        this.videoLink = this.domSantizer(`${this.videoLink}&rel=0&autoplay=${this.autoplay}`);
```

File: apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    ) { }
+
+    /**
+    * Initializes the component
+    *
+    * @memberof WatchVideoComponent
+    */
+    public ngOnInit(): void {
+        this.videoLink = VIDEOLINK[this.moduleName];
+        this.autoplay = this.autoplay ? 1 : 0;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts]: remove this line and take autoplay as 0/1

File: apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    /**
+    * Angular's sanitizer service to bypass security and trust the provided string as a resource URL
+    *
+    * @param {string} str
+    * @return {*}  {*}
+    * @memberof WatchVideoComponent
+    */
+    public domSantizer(str: string): any {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts:68]:
```suggestion
    public domSantizer(url: string): any {
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts]: ok

File: apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/theme/watch-video/watch-video.component.ts]: ok

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**